src/DataLib/fendl32B_retrofit/groupr_tools.py

```diff
@@ -91,9 +91,10 @@ def fill_input_template(material_id, MTs, element, A, mt_dict):
 
     card9_lines = []
     for MT in MTs:
-        mtname = mt_dict[MT]['Reaction']
-        MFD = 3 # ENDF file tag for cross-section data
-        card9_lines.append(f'{MFD} {MT} "{mtname}" /') 
+        if MT in mt_dict.keys():
+            mtname = mt_dict[MT]['Reaction']
+            MFD = 3 # ENDF file tag for cross-section data
```

Comment:
This can be set outside the loop since it doesn't change as you go through the loop